patches/minecraft/net/minecraft/world/WorldProvider.java.patch

```diff
@@ -118,19 +119,34 @@
 +        return 1.0;
 +    }
 +
++    @Deprecated
```

Comment:
Should add a note here why it is deprecated, and how the new system can be used instead.

Reply:
Okay.